src/Serializer/SubjectsAware.php

```diff
@@ -19,8 +20,11 @@
         $this->subjects = $subjects;
     }
 
-    final private function getSubjects(array $subjects) : PromiseInterface
+    final private function getSubjects($subjects) : PromiseInterface
```

Comment:
So like this: https://github.com/elifesciences/api-sdk-php/pull/20/commits/42b354b5343d1d69d878a465accf91d1675351b5 ?